core/src/main/java/com/datastax/oss/driver/internal/core/channel/ChannelFactory.java

```diff
@@ -149,31 +149,30 @@ private void connect(
 
     DriverConfigProfile config = context.config().getDefaultProfile();
 
-    boolean tcpNoDelay = config.getBoolean(DefaultDriverOption.CONNECTION_SOCKET_TCP_NODELAY);
```

Comment:
Noticed that some settings were moved up from `connection`, but others weren't, what was the thinking there?  Just to keep the full config path more brief or just to separate them from the rest of the connection options?  👍 in any case :)

Reply:
I moved `socket` out of `connection` to reduce the levels of nesting. `connection` is now only about the pooling / stream management aspects.